iceoryx_posh/test/moduletests/test_roudi_mempool_introspection.cpp

```diff
@@ -33,6 +33,26 @@ using ::testing::Return;
 #include "iceoryx_posh/roudi/introspection_types.hpp"
 #include "iceoryx_utils/cxx/vector.hpp"
 
+class CallChecker
+{
+  public:
+    MOCK_METHOD0(offer, void(void));
+};
+
+CallChecker& callChecker()
+{
+    static CallChecker cc;
+    return cc;
+}
+
+class MockPublisherPortUserAccess : public MockPublisherPortUser
+{
+  public:
+    void offer()
```

Comment:
call me pedantic, but if you have another commit, could you add `override` to the method

Reply:
Computer says: NO!